privacyidea/static/components/user/views/user.html

```diff
@@ -47,6 +47,25 @@
                                 translate>Edit realms</a>
                     </li>
                     <li translate>total users: {{ usercount }}</li>
+                    <hr>
+                    <li role="presentation"
+                        ng-show="($state.includes('user.list'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/webui/index.html#users'}}">
+                            <span class="glyphicon glyphicon-info-sign"></span>
+                            <span translate>Help about Users</span></a></li>
+                    <li role="presentation"
+                        ng-show="($state.includes('user.add'))">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/webui/manage_users.html#users'}}">
```

Comment:
No `#users# anchor here

Reply:
done in a37f2d98d2d6876d137ba6a6783b086b289941c3.